config/locales/en.yml

```
@@ -999,6 +999,8 @@ en:
               not_in_chargeable_state: "Invoice %{id} is not in chargeable state!"
             base:
               charging_failed: 'Failed to charge the credit card.'
+            period:
+              invalid_range: 'must be between the account creation date and 12 months from now'
```

IT would be useful to specify which account.

```suggestion
              invalid_range: 'must be between the provider account creation date and 12 months from now'
```

---

yeah... I thought about this, but I also feel like the term "provider" is kind of an internal one, which might be confusing to the customers. But well, we do use this term in the API docs, so maybe that's OK :)